content/get-started/importing-your-projects-to-github/importing-source-code-to-github/adding-locally-hosted-code-to-github.md

```diff
@@ -59,6 +59,13 @@ If you have existing source code or repositories stored locally on your computer
 {% data reusables.command_line.open_the_multi_os_terminal %}
 3. Change the current working directory to your local project.
 4. Initialize the local directory as a Git repository.
+   
+   {% warning %}
+
+    **Warning:**  `-b <branch-name>` flag in following command does not work with git version below *2.28.0*. Use same or above
+
+   {% endwarning %}
+   
   ```shell
   $ git init -b main
   ```
```

Comment:
If you’re using Git 2.27.1 or an earlier version, you can set the name of the default branch using  `&& git branch -m`.
  ```shell
  $ git init && git branch -m main
  ```